### PR TITLE
DOSCUP-6545: Edit and translate

### PR DESCRIPTION
**Pull request by olgarev**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #19267, 19297, 19527

### CODE REVIEW

File: docs/ru/engines/table-engines/mergetree-family/mergetree.md [hunk 1]
```diff
@@ -689,7 +689,101 @@ SETTINGS storage_policy = 'moving_from_ssd_to_hdd'
 
 Количество потоков для фоновых перемещений кусков между дисками можно изменить с помощью настройки [background_move_pool_size](../../../operations/settings/settings.md#background_move_pool_size)
 
-### Особенности работы {#osobennosti-raboty}
+## Использование сервиса S3 для хранения данных {#table_engine-mergetree-s3}
+
+Таблицы семейства `MergeTree` могут использовать для хранения данных сервис [S3](https://aws.amazon.com/s3/) с диском типа `s3`.
```
Comment (adevyatova) [inline docs/ru/engines/table-engines/mergetree-family/mergetree.md:694]:
```suggestion
Таблицы семейства `MergeTree` могут использовать для хранения данных сервиса [Amazon S3](https://aws.amazon.com/s3/) с диском типа `s3`.
```

File: docs/ru/engines/table-engines/mergetree-family/mergetree.md [hunk 2]
```diff
@@ -689,7 +689,101 @@ SETTINGS storage_policy = 'moving_from_ssd_to_hdd'
...
+
+Необязательные параметры:   
+
+-   `use_environment_credentials` — признак, нужно ли считывать учетные данные AWS из переменных окружения AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY и AWS_SESSION_TOKEN, если они есть. Значение по умолчанию: `false`.
+-   `proxy` — конфигурация прокси-сервера для конечной точки S3. Каждый элемент `uri` внутри блока `proxy` должен содержать URL прокси-сервера. 
+-   `connect_timeout_ms` — таймаут подключения к сокету в миллисекундах. Значение по умолчанию: 10 секунд. 
+-   `request_timeout_ms` — таймаут выполнения запроса в миллисекундах. Значение по умолчанию: 5 секунд. 
+-   `max_connections` — размер пула соединений S3. Значение по умолчанию: `100`. 
+-   `retry_attempts` — число попыток выполнения запроса в случае возникновения ошибки. Значение по умолчанию: `10`. 
+-   `min_bytes_for_seek` — минимальное количество байтов, которые используются для операций поиска вместо последовательного чтения. Значение по умолчанию: 1 Mb.
```
Comment (adevyatova) [inline docs/ru/engines/table-engines/mergetree-family/mergetree.md:740]:
```suggestion
-   `min_bytes_for_seek` — минимальное количество байтов, которые используются для операций поиска вместо последовательного чтения. Значение по умолчанию: 1 МБайт. 
```

File: docs/ru/engines/table-engines/mergetree-family/mergetree.md [hunk 3]
```diff
@@ -689,7 +689,101 @@ SETTINGS storage_policy = 'moving_from_ssd_to_hdd'
...
+                </external>
+            </volumes>
+            <move_factor>0.2</move_factor>
+        </s3_cold>
+    </policies>
+    ...
+</storage_configuration>
+```
+
+Если диск сконфигурирован как `cold`, данные будут переноситься в S3 либо когда свободное место на локальном диске станет меньше порогового значения, которое определяется как `move_factor * disk_size`, либо при срабатывании правил TTL.
```
Comment (adevyatova) [inline docs/ru/engines/table-engines/mergetree-family/mergetree.md:784]:
```suggestion
Если диск сконфигурирован как `cold`, данные будут переноситься в S3 при срабатывании правил TTL, либо когда свободное место на локальном диске станет меньше порогового значения, которое определяется как `move_factor * disk_size`. 
```

File: docs/ru/operations/utilities/clickhouse-local.md [hunk 4]
```diff
@@ -31,15 +31,23 @@ $ clickhouse-local --structure "table_structure" --input-format "format_of_incom
...
 -   `--verbose` — подробный вывод при выполнении запроса.
--   `-s` — отключает вывод системных логов в `stderr`.
--   `--config-file` — путь к файлу конфигурации. По умолчанию `clickhouse-local` запускается с пустой конфигурацией. Конфигурационный файл имеет тот же формат, что и для сервера ClickHouse и в нём можно использовать все конфигурационные параметры сервера. Обычно подключение конфигурации не требуется, если требуется установить отдельный параметр, то это можно сделать ключом с именем параметра.
+-   `--logger.console` — логирование действий в консоль.
+-   `--logger.log` — логирование действий в файл с указанным именем.
+-   `--logger.level` — уровень логирования.
+-   `--ignore-error` — не прекращать обработку если запрос выдал ошибку.
+-   `-c`, `--config-file` — путь к файлу конфигурации. По умолчанию `clickhouse-local` запускается с пустой конфигурацией. Конфигурационный файл имеет тот же формат, что и для сервера ClickHouse и в нём можно использовать все конфигурационные параметры сервера. Обычно подключение конфигурации не требуется, если требуется установить отдельный параметр, то это можно сделать ключом с именем параметра.
```
Comment (adevyatova) [inline docs/ru/operations/utilities/clickhouse-local.md:45]:
```suggestion
-   `-c`, `--config-file` — путь к файлу конфигурации. По умолчанию `clickhouse-local` запускается с пустой конфигурацией. Конфигурационный файл имеет тот же формат, что и для сервера ClickHouse, и в нём можно использовать все конфигурационные параметры сервера. Обычно подключение конфигурации не требуется; если требуется установить отдельный параметр, то это можно сделать ключом с именем параметра.
```

**Comment by olgarev** [comment]
@vdimir Hi, will you review this PR, please? 

Concerning the "Using S3 for Data Storage" section. 
This is an "Edit and translate" ticket, so I've just translated the english text. But are you sure the section about S3 is in the right place? Seems that the next "Details" section (it has level <h3>) is not related to the S3 section (level <h2>). May be S3 section should be moved to the end of the page, after "Details"?

File: docs/ru/engines/table-engines/mergetree-family/mergetree.md [hunk 5]
```diff
@@ -689,7 +689,101 @@ SETTINGS storage_policy = 'moving_from_ssd_to_hdd'
 
 Количество потоков для фоновых перемещений кусков между дисками можно изменить с помощью настройки [background_move_pool_size](../../../operations/settings/settings.md#background_move_pool_size)
 
-### Особенности работы {#osobennosti-raboty}
+## Использование сервиса S3 для хранения данных {#table_engine-mergetree-s3}
+
+Таблицы семейства `MergeTree` могут использовать для хранения данных сервиса [Amazon S3](https://aws.amazon.com/s3/) с диском типа `s3`.
```
Comment (nikitamikhaylov) [inline docs/ru/engines/table-engines/mergetree-family/mergetree.md:694]:
```suggestion
Таблицы семейства `MergeTree` могут хранить данные в сервисе [S3](https://aws.amazon.com/s3/) при использовании диска типа `s3`.
```

File: docs/ru/engines/table-engines/mergetree-family/mergetree.md [hunk 6]
```diff
@@ -689,7 +689,101 @@ SETTINGS storage_policy = 'moving_from_ssd_to_hdd'
...
+            <skip_access_check>false</skip_access_check>
+        </s3>
+    </disks>
+    ...
+</storage_configuration>
+```
+
+Обязательные параметры:
+
+-   `endpoint` — URL конечной точки S3 в запросах `path` или `virtual hosted` [styles](https://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html). URL конечной точки должен содержать бакет и путь к корневой директории на сервере, где хранятся данные.
```
Comment (nikitamikhaylov) [inline docs/ru/engines/table-engines/mergetree-family/mergetree.md:728]:
```suggestion
-   `endpoint` — URL точки приема запроса на стороне S3 в [форматах](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html) `path` или `virtual hosted`. URL точки должен содержать бакет и путь к корневой директории на сервере, где хранятся данные.
```

File: docs/ru/engines/table-engines/mergetree-family/mergetree.md [hunk 7]
```diff
@@ -689,7 +689,101 @@ SETTINGS storage_policy = 'moving_from_ssd_to_hdd'
...
+-   `proxy` — конфигурация прокси-сервера для конечной точки S3. Каждый элемент `uri` внутри блока `proxy` должен содержать URL прокси-сервера. 
+-   `connect_timeout_ms` — таймаут подключения к сокету в миллисекундах. Значение по умолчанию: 10 секунд. 
+-   `request_timeout_ms` — таймаут выполнения запроса в миллисекундах. Значение по умолчанию: 5 секунд. 
+-   `max_connections` — размер пула соединений S3. Значение по умолчанию: `100`. 
+-   `retry_attempts` — число попыток выполнения запроса в случае возникновения ошибки. Значение по умолчанию: `10`. 
+-   `min_bytes_for_seek` — минимальное количество байтов, которые используются для операций поиска вместо последовательного чтения. Значение по умолчанию: 1 МБайт. 
+-   `metadata_path` — путь к локальному файловому хранилищу для хранения файлов с метаданными для S3. Значение по умолчанию: `/var/lib/clickhouse/disks/<disk_name>/`. 
+-   `cache_enabled` — признак, разрешено ли хранение меток кеша и индексных файлов в локальном файловом хранилище. Значение по умолчанию: `true`.
```
Comment (nikitamikhaylov) [inline docs/ru/engines/table-engines/mergetree-family/mergetree.md:742]:
```suggestion
-   `cache_enabled` — признак, разрешено ли хранение кэша засечек и файлов индекса в локальной файловой системе. Значение по умолчанию: `true`. 
```

File: docs/ru/engines/table-engines/mergetree-family/mergetree.md [hunk 8]
```diff
@@ -689,7 +689,101 @@ SETTINGS storage_policy = 'moving_from_ssd_to_hdd'
...
+-   `connect_timeout_ms` — таймаут подключения к сокету в миллисекундах. Значение по умолчанию: 10 секунд. 
+-   `request_timeout_ms` — таймаут выполнения запроса в миллисекундах. Значение по умолчанию: 5 секунд. 
+-   `max_connections` — размер пула соединений S3. Значение по умолчанию: `100`. 
+-   `retry_attempts` — число попыток выполнения запроса в случае возникновения ошибки. Значение по умолчанию: `10`. 
+-   `min_bytes_for_seek` — минимальное количество байтов, которые используются для операций поиска вместо последовательного чтения. Значение по умолчанию: 1 МБайт. 
+-   `metadata_path` — путь к локальному файловому хранилищу для хранения файлов с метаданными для S3. Значение по умолчанию: `/var/lib/clickhouse/disks/<disk_name>/`. 
+-   `cache_enabled` — признак, разрешено ли хранение меток кеша и индексных файлов в локальном файловом хранилище. Значение по умолчанию: `true`. 
+-   `cache_path` — путь к локальному файловому хранилищу, где хранятся метки кеша и индексные файлы. Значение по умолчанию: `/var/lib/clickhouse/disks/<disk_name>/cache/`.
```
Comment (nikitamikhaylov) [inline docs/ru/engines/table-engines/mergetree-family/mergetree.md:743]:
```suggestion
-   `cache_path` — путь в локальной файловой системе, где будут храниться кэш засечек и файлы индекса. Значение по умолчанию: `/var/lib/clickhouse/disks/<disk_name>/cache/`. 
```

File: docs/ru/engines/table-engines/mergetree-family/mergetree.md [hunk 9]
```diff
@@ -689,7 +689,101 @@ SETTINGS storage_policy = 'moving_from_ssd_to_hdd'
...
+-   `request_timeout_ms` — таймаут выполнения запроса в миллисекундах. Значение по умолчанию: 5 секунд. 
+-   `max_connections` — размер пула соединений S3. Значение по умолчанию: `100`. 
+-   `retry_attempts` — число попыток выполнения запроса в случае возникновения ошибки. Значение по умолчанию: `10`. 
+-   `min_bytes_for_seek` — минимальное количество байтов, которые используются для операций поиска вместо последовательного чтения. Значение по умолчанию: 1 МБайт. 
+-   `metadata_path` — путь к локальному файловому хранилищу для хранения файлов с метаданными для S3. Значение по умолчанию: `/var/lib/clickhouse/disks/<disk_name>/`. 
+-   `cache_enabled` — признак, разрешено ли хранение меток кеша и индексных файлов в локальном файловом хранилище. Значение по умолчанию: `true`. 
+-   `cache_path` — путь к локальному файловому хранилищу, где хранятся метки кеша и индексные файлы. Значение по умолчанию: `/var/lib/clickhouse/disks/<disk_name>/cache/`. 
+-   `skip_access_check` — признак, выполнять ли проверку доступа к диску при запуске диска. Если установлено значение `true`, то проверка не выполняется. Значение по умолчанию: `false`.
```
Comment (nikitamikhaylov) [inline docs/ru/engines/table-engines/mergetree-family/mergetree.md:744]:
```suggestion
-   `skip_access_check` — признак, выполнять ли проверку доступов при запуске диска. Если установлено значение `true`, то проверка не выполняется. Значение по умолчанию: `false`.
```

File: docs/en/sql-reference/functions/math-functions.md [hunk 10]
```diff
@@ -415,7 +415,7 @@ Result:
 
 ## sign(x) {#signx}
 
-The `sign` function can extract the sign of a real number.
+Extracts the sign of a real number.
```
Comment (nikitamikhaylov) [inline docs/en/sql-reference/functions/math-functions.md:418]:
```suggestion
Returns the sign of a real number.
```

File: docs/ru/sql-reference/functions/math-functions.md [hunk 11]
```diff
@@ -405,4 +405,67 @@ SELECT log1p(0);
 
 -   [log(x)](../../sql-reference/functions/math-functions.md#logx)
 
+## sign(x) {#signx}
+
+Выделяет знак действительного числа.
```
Comment (nikitamikhaylov) [inline docs/ru/sql-reference/functions/math-functions.md:410]:
```suggestion
Возвращает знак действительного числа.
```

File: docs/ru/sql-reference/functions/math-functions.md [hunk 12]
```diff
@@ -405,4 +405,67 @@ SELECT log1p(0);
...
+
+Результат:
+
+``` text
+┌─sign(1)─┐
+│       1 │
+└─────────┘
+```
+
+Знак для отрицательного значения:
```
Comment (nikitamikhaylov) [inline docs/ru/sql-reference/functions/math-functions.md:457]:
```suggestion
Результат sign() для отрицательного аргумента:
```

File: docs/ru/sql-reference/functions/math-functions.md [hunk 13]
```diff
@@ -405,4 +405,67 @@ SELECT log1p(0);
...
+```
+Результат:
+
+``` text
+┌─sign(0)─┐
+│       0 │
+└─────────┘
+```
+
+Знак для положительного значения:
```
Comment (nikitamikhaylov) [inline docs/ru/sql-reference/functions/math-functions.md:443]:
```suggestion
Результат sign() для положительного аргумента:
```

File: docs/ru/sql-reference/functions/math-functions.md [hunk 14]
```diff
@@ -405,4 +405,67 @@ SELECT log1p(0);
...
+
+**Возвращаемое значение**
+
+- -1 если `x < 0`
+-  0 если `x = 0`
+-  1 если `x > 0`
+
+**Примеры**
+
+Знак для нулевого значения:
```
Comment (nikitamikhaylov) [inline docs/ru/sql-reference/functions/math-functions.md:430]:
```suggestion
Результат sign() для нуля:
```

**Review by nikitamikhaylov — CHANGES_REQUESTED** [review]

**Comment by olgarev** [comment]
@nikitamikhaylov Thanks a lot for review, I applied all suggestions. What about "Details" section (see my question above)?

**Comment by olgarev** [comment]
@nikitamikhaylov Headers corrected.

**Review by nikitamikhaylov — APPROVED** [review]

**Pull request merged**